CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # TiDB Changelog
 All notable changes to this project will be documented in this file. See also [Release Notes](https://github.com/pingcap/docs/blob/master/releases/rn.md), [TiKV Changelog](https://github.com/tikv/tikv/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.1.6] 2019-03-15
+
+### SQL Optimizer/Executor
+* Optimize planner to select the outer table based on cost when both tables are specified in Hint of `TIDB_INLJ` [#9615](https://github.com/pingcap/tidb/pull/9615)
+* Fix the issue when IndexScan cannot be selected correctly in some cases
+[#9587](https://github.com/pingcap/tidb/pull/9587)
+* Fix incompatibility with MySQL of check in the `agg` function in subqueries  [#9551](https://github.com/pingcap/tidb/pull/9551)
+* Fix the issue when `show stats_histogram` only outputs valid columns to avoid kernel panics
+ [#9502](https://github.com/pingcap/tidb/pull/9502)
+### Server
+* Support variable `log_bin` to enable/disable Binlog [#9634](https://github.com/pingcap/tidb/pull/9634)
+* Add a sanity check for transactions to avoid false transaction commit [#9559](https://github.com/pingcap/tidb/pull/9559)
+* Fix the issue when setting a variable with control characters causes a kernel panic  [#9539 ](https://github.com/pingcap/tidb/pull/9539)
+
+### DDL
+* Fix the issue when the `Create Table Like` statement causes a panic issue  [#9652](https://github.com/pingcap/tidb/pull/9652)
+* Enable AutoSync for etcd clients to avoid connection issues between TiDB and etcd in some cases [#9600](https://github.com/pingcap/tidb/pull/9600)
```

Comment:
```suggestion
* Enable the `AutoSync` feature of etcd clients to avoid connection issues between TiDB and etcd in some cases [#9600](https://github.com/pingcap/tidb/pull/9600)
```